packages/server/test/unit/modes/record_spec.js

```diff
@@ -439,27 +439,24 @@ describe('lib/modes/record', () => {
       })
     })
 
-    it('does not createException when statusCode is 503', () => {
+    it('errors when statusCode is 503', async () => {
       const err = new Error('foo')
 
       err.statusCode = 503
 
-      sinon.spy(logger, 'createException')
+      sinon.spy(errors, 'get')
 
       sinon.stub(api, 'retryWithBackoff').rejects(err)
 
-      return recordMode.createInstance({
+      await expect(recordMode.createInstance({
         runId: 'run-123',
         groupId: 'group-123',
         machineId: 'machine-123',
         platform: {},
         spec: { relative: 'cypress/integration/app_spec.coffee' },
-      })
-      .then((ret) => {
-        expect(ret).to.be.null
+      })).to.be.rejected
```

Comment:
looks like we lost the `expect(ret).to.be.null` assertion

Reply:
that's intended as this method no longer warns in serial mode, it now throws